src/main/java/cart/service/MemberService.java

```diff
@@ -15,7 +16,7 @@
 
 @Service
 public class MemberService {
-    private final H2MemberDao memberDao;
+    private final MemberDao memberDao;
     private final MemberMapper memberMapper;
 
     public MemberService(H2MemberDao memberDao, MemberMapper memberMapper) {
```

Comment:
객체 주입도 추상화를 주입해주어야 합니다아~~!
```suggestion
    public MemberService(MemberDao memberDao, MemberMapper memberMapper) {
```